mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/ReactorRabbitMQChannelPoolTest.java

```diff
@@ -81,7 +89,24 @@ void concurrentRequestOnChannelMonoLeadToPoolWaitTimeoutException() {
                 .runSuccessfullyWithin(Duration.ofSeconds(30)))
             .isInstanceOf(ExecutionException.class)
             .hasMessageContaining("java.util.NoSuchElementException: Timeout waiting for idle object");
+    }
 
-        listChannels.forEach(Throwing.consumer(Channel::close));
+    @Test
+    void usedChannelShouldBeClosedWhenPoolIsClosed() {
+        ChannelPool channelPool = generateChannelPool(2);
+        Channel channel = channelPool.getChannelMono().block();
+        assertThat(channel.isOpen()).isTrue();
```

Comment:
Do you need this assert ?

Reply:
In theory no but it serves two purposes:

 1. Ensure that we get a sound Channel
 2. Show that we expect our behavior to close it